refreeze.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+echo 'Refreezing...'
+deepest_level=$(find . -name 'package.dhall' | sed 's/[^\/]//g' | awk '{ print length }' | sort | tail -n 1)
+for depth in $(seq "${deepest_level}" -1 1) ; do
+  while IFS= read -r -d '' package_dhall ; do
+    echo "Freezing: ${package_dhall} ..."
+    dhall freeze --cache --all --inplace "${package_dhall}"
+  done < <(find . -mindepth "${depth}" -maxdepth "${depth}" -name 'package.dhall' -print0)
+done
+echo 'Finished refreezing successfully!'
```

Comment:
Should change to include also `dhall format` and `dhall <<<`

Also, README.md is a requirement and it should include all the relevant information how to use the script.

Reply:
Why do you have both `refreeze` and `freeze`?

Reply:
The dhall command is `freeze` but as all of the packages were already frozen (to the wrong imports) they need to be re-frozen. If something is already frozen, to the naive ear, it sounds weird to freeze something already frozen.